src/lib/components/dominant_color/image-worker.ts

```diff
@@ -10,10 +10,10 @@ onmessage = async function (event) {
 		dominant_color =
 			chroma(color_palette[0]).get("lab.l") < 40
 				? chroma(color_palette[0]).brighten().hex()
-				: chroma(color_palette[0]).hex(),
+				: chroma(color_palette[0]).darken(2).hex(),
 		dominant_foreground_color =
 			chroma(color_palette[0]).get("lab.l") < 40
-				? chroma(color_palette[0]).brighten(2).hex()
+				? chroma(color_palette[0]).brighten(3).hex()
 				: chroma(color_palette[0]).hex();
```

Comment:
This is being touched too much

Reply:
Yea, well we need to return variety of shades of dominant color.
return more shades?

Reply:
I do understand it, but i feel like you are touching it too much. Maybe use chroma  for each item?

Reply:
> Maybe use chroma for each item?

You want to use chrome outside?

Reply:
> You want to use chrome outside?

Sure